lib/stages/VisSharedMemWriter.hpp

```diff
@@ -0,0 +1,105 @@
+#ifndef VISSHAREDMEMWRITER_HPP
```

Comment:
This needs a module level docstring.

Reply:
This is probably the correct place to add a write up of how the buffer scheme works and how all the data is packed, etc.